tables.md

```diff
@@ -0,0 +1,198 @@
+| **Notable fields:** |
+| `bootstrap_elapsed_secs` | The length of the bootstrap. |
+| `num_fatals` | Number of fatal severity log lines during the session. |
+| `num_errors` | Number of error severity log lines during the session. |
+| `num_warnings` | Number of warning severity log lines during the session. |
+
+| `p_load_dates` ||
+| :---: | --------- |
+| | *Technical table about Palette internals* |
+
+| `p_serverlogs` ||
+| :---: | --------- |
+| | Loaded from the serverlogs table. This table contains process relationship information. (parent_vizqlsession, parent_dataserver_session etc) and some smaller data cleanup like domain is removed from username and such.|
+| **Notable fields:** |
+| `filename` | The original filename of the log file from which the given line is |
+| `sev` | Severity of the current log line |
+| `k` | Key (or category) of the current log line |
+| `v` | Actual content of the currnet log line |
+| `elapsed_ms` | Parsed duration of the event descibed in this log line |
+
+| `p_serverlogs_bootstrap_rpt` ||
+| :---: | --------- |
+| | Has the same data as p_serverlogs but limited to records that happened before the Bootstrap action in Tableau has ended.|
+| **Notable fields:** |
+| `session_start_ts` | The UTC timestamp of the beginning of the session. |
+| `session_duration` | How long the actual session lasted. |
+| `currentsheet` | Which sheet of the workbook was originally opened. (During the bootstrap.) |
+| `session_elapsed_seconds` | How long the session bootstrap has been taking in the time of this measurement. |
+
+| `p_threadinfo` ||
+| :---: | --------- |
+| | *Currently not used* |
+
+| `p_threadinfo_delta` ||
+| :---: | --------- |
+| | Based on threadinfo raw table. Contains the CPU consumption between two measures in threadinfo for the same process / thread. For the same process / thread there is always exactly 15 seconds time difference between two measures. |
+| **Notable fields:** |
+| `process_id ` | Exact id of the process. This is unique for a given time for a given machine. |
+| `thread_id` | id of the thread if the current measurement is referring to a thread, `-1` otherwise. (`-1` means the current row refers to the whole process.) |
+| `cpu_core_consumption` | The actual cpu usage that happened for this record since the previous measurement. |
+| `memory_usage_bytes` | The actual memory usage in bytes for this record. |
+| `ts_rounded_15_secs` | The timestamp the measurement took place rounded to a 15 seconds polling interval. |
+
+| `plainlogs` ||
+| :---: | --------- |
+| | There two kind of Tableau Server log files. One set of processes log in JSON format and an other set of them just in plain text files. This table contains the raw log lines from plain text log files. |
```

Comment:
I think it was meant to start like: "There are two kind of ...".
And I don't think I get the second sentence. I guess it was meant to start like: "One set of them processes(? - really? not contains or something else? processes is the right word here?)"

... and I would continue like this: "... and an other set of them is just plain text in files."

Reply:
There are two kinds of Tableau Server log files. One gang, band, group, SET of the processes of that SoaB Tableu are logging in JSON and the other pals log in plain text.